### PR TITLE
WIP: increase website score

### PR DESCRIPTION
**Pull request by harsh-vador**

### CODE REVIEW

**Review by Sachin-chaurasiya — APPROVED** [review]

File: components/ConnectorsInfo/ConnectorImage.tsx [hunk 1]
```diff
@@ -10,5 +11,14 @@ export default function ConnectorImage(
...
+  return (
+    <Image
+      width={42}
+      height={42}
+      className="w-42 h-42"
+      src={props.src}
+      alt={props.alt}
+      onError={replaceImgWithError}
+    />
+  );
```
Comment (aniketkatkar97) [inline components/ConnectorsInfo/ConnectorImage.tsx:23]: We are giving both width height and relevant className here. Any reason we are giving both? Also previously we could give custom props to the images, we're not allowing that now.

File: components/common/Youtube/Youtube.tsx [hunk 2]
```diff
@@ -1,27 +1,18 @@
...
-        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-        style={{ height, width }}
-      ></iframe>
-    </div>
+    <>
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+      </Head>
+
+      <LiteYouTubeEmbed webp={true} id={videoId} title="YouTube Video Player" />
```
Comment (aniketkatkar97) [inline components/common/Youtube/Youtube.tsx:14]: Removal of all the styling and props has affected the youtube video stylings all over the documentation. Can we please restore the previous styling along with these changes?

File: components/ConnectorsInfo/ConnectorsInfo.tsx [hunk 3]
```diff
@@ -1,16 +1,16 @@
 import classNames from "classnames";
-import { uniqBy } from "lodash";
+import {uniqBy} from "lodash";
 import dynamic from "next/dynamic";
 import Link from "next/link";
-import { useState } from "react";
-import { useDocVersionContext } from "../../context/DocVersionContext";
+import {useState} from "react";
+import {useDocVersionContext} from "../../context/DocVersionContext";
```
Comment (aniketkatkar97) [inline components/ConnectorsInfo/ConnectorsInfo.tsx:6]: Can we please avoid these formatting changes? these are all over the place

File: pages/_app.tsx [hunk 4]
```diff
@@ -6,23 +6,23 @@ import "prismjs";
...
-import { MenuItemsContextProvider } from "../context/MenuItemsContext";
-import { NavBarCollapseContextProvider } from "../context/NavBarCollapseContext";
-import { RouteChangingContextProvider } from "../context/RouteChangingContext";
-import { StepsContextProvider } from "../context/StepsContext";
+import {CodeWithLanguageSelectorContextProvider} from "../context/CodeWithLanguageSelectorContext";
+import {DocVersionContextProvider} from "../context/DocVersionContext";
+import {MenuItemsContextProvider} from "../context/MenuItemsContext";
+import {NavBarCollapseContextProvider} from "../context/NavBarCollapseContext";
+import {RouteChangingContextProvider} from "../context/RouteChangingContext";
+import {StepsContextProvider} from "../context/StepsContext";
```
Comment (aniketkatkar97) [inline pages/_app.tsx:17]: Same here, these formatting changes are unnecessary and some of the files only have these.

**Review by aniketkatkar97 — COMMENTED** [review]
Left some comments

File: components/ConnectorsInfo/ConnectorImage.tsx (hunk 1 above)
Comment (harsh-vador) [reply components/ConnectorsInfo/ConnectorImage.tsx]: This is being handled now

File: components/common/Youtube/Youtube.tsx (hunk 2 above)
Comment (harsh-vador) [reply components/common/Youtube/Youtube.tsx]: Revert these changes and added lazy loading

**Review by aniketkatkar97 — APPROVED** [review]

**Pull request closed without merging**